### ISSUE TITLE
Is project alive ?

### ISSUE THREAD

**Issue by DisasteR** [claudyus/LXC-Web-Panel#83]
Hi,

Didn't Saw any change since à while. Is this project still alive ?

Regards

**Comment by claudyus** [comment]
Yes the project is still alive and I'm happy to accepts PR or maintainers if anyone is interested.

The effort in developing new features is decreased now that LXD is more reliable alternative to lxclite command line wrapper. Some PR in this direction should be really appreciated.

**Comment by fossxplorer** [comment]
@claudyus so if we want to create any integration with LXC web panel as basis we can be sure it will live for many years to come?
I'm looking for a VM manager as replacement for HyperVM and this might be it. Now it needs integration with Boxbilling and the likes :)
